### ISSUE TITLE
PHP8.0.1 building error

### ISSUE THREAD

**Issue by damixxx** [zephir-lang/zephir#2130]
Hello. I red all of the posted issues about PHP8 but it is not very clear for me if zephir is compatible with PHP8. I updated zephir to version 0.12.20 and zephir-parser to 1.3.6. The extension is building fine with PHP7.2. Whit PHP8.0.1 I get:

```
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:56:71: error: expected ‘;’, ‘,’ or ‘)’ before ‘TSRMLS_DC’
 static void php_zephir_init_globals(zend_splab_globals *splab_globals TSRMLS_DC)
                                                                       ^~~~~~~~~
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:76:78: error: expected ‘;’, ‘,’ or ‘)’ before ‘TSRMLS_DC’
 static void php_zephir_init_module_globals(zend_splab_globals *splab_globals TSRMLS_DC)
                                                                              ^~~~~~~~~
/home/ubuntu/zephirDevelopment/splab/ext/splab.c: In function ‘zm_activate_splab’:
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:86:2: warning: implicit declaration of function ‘php_zephir_init_globals’; did you mean ‘zephir_get_global’? [-Wimplicit-function-declaration]
  php_zephir_init_globals(splab_globals_ptr);
  ^~~~~~~~~~~~~~~~~~~~~~~
  zephir_get_global
/home/ubuntu/zephirDevelopment/splab/ext/splab.c: In function ‘zm_deactivate_splab’:
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:96:29: error: ‘TSRMLS_C’ undeclared (first use in this function); did you mean ‘TSRM_H’?
  zephir_deinitialize_memory(TSRMLS_C);
                             ^~~~~~~~
                             TSRM_H
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:96:29: note: each undeclared identifier is reported only once for each function it appears in
/home/ubuntu/zephirDevelopment/splab/ext/splab.c: In function ‘zm_globals_ctor_splab’:
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:126:2: warning: implicit declaration of function ‘php_zephir_init_module_globals’; did you mean ‘zephir_get_global’? [-Wimplicit-function-declaration]
  php_zephir_init_module_globals(splab_globals);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  zephir_get_global
make: *** [splab.lo] Error 1
make: *** Waiting for unfinished jobs....
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:56:71: error: expected ‘;’, ‘,’ or ‘)’ before ‘TSRMLS_DC’
 static void php_zephir_init_globals(zend_splab_globals *splab_globals TSRMLS_DC)
                                                                       ^~~~~~~~~
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:76:78: error: expected ‘;’, ‘,’ or ‘)’ before ‘TSRMLS_DC’
 static void php_zephir_init_module_globals(zend_splab_globals *splab_globals TSRMLS_DC)
                                                                              ^~~~~~~~~
/home/ubuntu/zephirDevelopment/splab/ext/splab.c: In function ‘zm_activate_splab’:
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:86:2: warning: implicit declaration of function ‘php_zephir_init_globals’; did you mean ‘zephir_get_global’? [-Wimplicit-function-declaration]
  php_zephir_init_globals(splab_globals_ptr);
  ^~~~~~~~~~~~~~~~~~~~~~~
  zephir_get_global
/home/ubuntu/zephirDevelopment/splab/ext/splab.c: In function ‘zm_deactivate_splab’:
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:96:29: error: ‘TSRMLS_C’ undeclared (first use in this function); did you mean ‘TSRM_H’?
  zephir_deinitialize_memory(TSRMLS_C);
                             ^~~~~~~~
                             TSRM_H
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:96:29: note: each undeclared identifier is reported only once for each function it appears in
/home/ubuntu/zephirDevelopment/splab/ext/splab.c: In function ‘zm_globals_ctor_splab’:
/home/ubuntu/zephirDevelopment/splab/ext/splab.c:126:2: warning: implicit declaration of function ‘php_zephir_init_module_globals’; did you mean ‘zephir_get_global’? [-Wimplicit-function-declaration]
  php_zephir_init_module_globals(splab_globals);
  ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  zephir_get_global
make: *** [splab.lo] Error 1
```

**Comment by Jeckerson** [comment]
Zephir isn't compatibly with PHP8 atm, stay tuned.

**Comment by Jeckerson** [comment]
Closing in favor #2111
